src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -94,22 +94,16 @@ Developer tools for Firefox for Android are provided through remote debugging me
 
 ### Effect on your add-on UI
 
-You cannot reveal your add-on through a sidebar or context menu. However, you can expose your extension as an option under the Add-ons item in the browser menu or as an address bar button.
+#### Reveal your add-on
 
-Depending on the manifest version used by your app, you add an option under the Add-ons item in the browser menu:
+You reveal your add-on as an option under the Add-ons item in the Firefox for Android browser menu. Depending on the manifest version used by your app, you add an option like this:
 
 - for Manifest V2 with the `manifest.json` [`browser_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action) key and [`browserAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/browserAction) API.
 - for Manifest V3 with the `manifest.json` [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/action) key and [`action`](https://developer.mozilla.org//Add-ons/WebExtensions/API/action) API.
 
-You can also use an address bar button (through the `manifest.json` [`page_action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/page_action) key and [`pageAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/pageAction) API), remembering that this button is hidden by default and must be shown programmatically.
+Note that the [`browserAction`](https://developer.mozilla.org//Add-ons/WebExtensions/API/browserAction) and [`action`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/action) popup content opens as an overlay, covering the browser window until the user closes the overlay.
 
-The features of [`pageAction`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/pageAction) in Firefox for Android differ slightly from the desktop version. 
```

Comment:
This part can be kept, as there is still a difference: on desktop, the pageAction button may appear inside the address bar, but on mobile, the button appears in the same place as the browserAction (MV2)/action (MV3) button.